.github/renovate.json

```diff
@@ -18,6 +17,18 @@
     "github.com/imdario/mergo"
   ],
   "packageRules": [
+    {
+      "matchBaseBranches": ["main"],
+      "extends": ["github>rancher/renovate-config:rancher-main#main"]
+    },
+    {
+      "matchBaseBranches": ["release/v6.x"],
+      "extends": ["github>rancher/renovate-config:rancher-2.10#main"]
+    },
+    {
+      "matchBaseBranches": ["release/v5.0"],
+      "extends": ["github>rancher/renovate-config:rancher-2.9#main"]
+    },
     {
       "groupName": "GitHub Workflow Actions",
```

Comment:
Mostly as a "by the bye", both group slugs could be defined in `renovate-config`, so that all projects benefit from it.